README.md

```diff
@@ -1,2 +1,35 @@
-# working-group-template
-Template for starting and maintaining a working group or team in The Turing Way
+# The Turing Way Book Dash 📚💨
+
+This repository is being used to document processes and decisions made for *The Turing Way* Book Dash.
+
+The Book Dash is a week long hackathon where participants collaboratively work on *The Turing Way* book synchronously to develop new chapters and review/edit existing ones to make them more accessible, comprehensive and up-to-date. 
+Participants also contribute to enhancing the project by improving the ways we work in the community and take the lead on accomplishing different tasks or sub-projects.
+
+Find out more about the Book Dash [here](https://the-turing-way.netlify.app/community-handbook/bookdash).
+
+![A person helping out another person making their first pull request on GitHub](https://github.com/the-turing-way/bookdash/first-pull-request.svg "First pull request")
+
+## Book Dash Working Group
+
+The Book Dash Working Group was initiated in January 2024. 
+It's purpose is to make long term strategic decisions about the Book Dash events - to be decided and agreed in a future meeting.
+Members of the working group will serve for at least one year and need to have previous experience of being on the Book Dash planning committee.
+
+### Current members of the working group
+
+* Esther Plomp - Chair
+* Arielle Bennett - Secretary
+* Emma Karoune - Secretary
+* Carlos Martinez
+* Susana Roman Garcia
+
+Our working group meeting notes can be found [here](https://github.com/the-turing-way/bookdash/wg-meeting-notes)
+
+## Book Dash Planning Committee
+Current members of our community that have previously joined a Book Dash can apply to be on the planning committee for an individual Book Dash event.
+
```

Comment:
At least one Book Dash Working Group member will attend planning committee meetings to support communication between the two groups.